rotkehlchen/tests/db/test_db_upgrades.py

```diff
@@ -2025,6 +2025,16 @@ def test_upgrade_db_40_to_41(user_data_dir):
             assert names[0] in should_move_used_query_ranges | should_not_move_used_query_ranges
         cursor.execute('SELECT COUNT(*) FROM location WHERE location=? AND seq=?', ('m', 45))
         assert cursor.fetchone()[0] == 0
+        assert cursor.execute('SELECT * FROM blockchain_accounts;').fetchall() == [
```

Comment:
Also for this DB upgrade test you need:

1. To test that the `blockchain_accounts` table exists before with the old schema.
2. Run it 3 times. One as is. One with a customized setting to the DB for the priority. One with a broken setting in the DB for the order. And see it works fine for all cases.